ckanext/data_qld/plugin.py

```diff
@@ -107,6 +112,26 @@ def before_map(self, m):
                   controller='ckanext.data_qld.controller:DataQldUI',
                   action='show_schema', conditions=dict(method=['GET']))
 
+        # This is a pain, but re-assigning the dataset_read route using `before_map` appears to affect these two routes,
```

Comment:
with that comment, should we at a later date try and incorporate this into the ckan core by pull-request and once accepted and we move to that next version, remove this from our current extension? (backlog ticket me thinks)

Reply:
yeap, agreed - as CKAN continues to migrate the current controllers to blueprints, this solution for overriding dataset/package URLs will no longer work

Reply:
if time is available, please raise an issue with the main ckan project and link to this comment as if we are having issues, many others should be running into it also. there may be a better way that the ckan core can be tweaked so that simple extensions like this does not require overriding it all manually.